### PR TITLE
add delivery-add-command with parser

### LINKED ISSUE #96: Add implementation for delivery

### LINKED ISSUE #132: Check isSameDelivery method if its necessary (refactoring)

### PR DESCRIPTION
**Pull request by bobbylaw**
- [x] Add implementation for delivery
- [x] Remove isSameDelivery and uses equals for checks
- [x] Command & Parser testcase
- [x] update UG/DG on delivery add 
- [x] Delivery Storage testcase

### CODE REVIEW

**Comment by Wincenttjoi** [comment]
> things to be added
> 
> * Command & Parser testcase

Are you planning to add it in this PR or another PR?

**Comment by bobbylaw** [comment]
I will be adding it into this later tonight

**Comment by Wincenttjoi** [comment]
> I will be adding it into this later tonight

@zeranium97 Ok, normally a common practice that people use if the PR is not ready they will use WIP in the header. Changed for you! **Ofc the CI needs to be put first and I just made a PR for that

**Comment by bobbylaw** [comment]
Alrights, Thanks @Wincenttjoi

File: src/main/java/seedu/address/logic/commands/deliverycommand/DeliveryAddCommand.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+ */
+public class DeliveryAddCommand extends DeliveryCommand {
+
+    public static final String COMMAND_WORD = "add-d";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a delivery to the delivery book. "
+            + "Parameters: "
+            + PREFIX_NAME + "NAME "
+            + PREFIX_PHONE + "QUANTITY "
+            + PREFIX_ADDRESS + "SUPPLIER "
```
Comment (Choo-Xing-Yu) [inline src/main/java/seedu/address/logic/commands/deliverycommand/DeliveryAddCommand.java]: Should be address, not supplier

Comment (Wincenttjoi) [reply src/main/java/seedu/address/logic/commands/deliverycommand/DeliveryAddCommand.java]: And should be phone, not quantity! Hahah

Comment (Wincenttjoi) [inline src/main/java/seedu/address/logic/commands/deliverycommand/DeliveryAddCommand.java]:
```suggestion
            + PREFIX_ADDRESS + "ADDRESS "
```

File: src/main/java/seedu/address/logic/commands/deliverycommand/DeliveryAddCommand.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+ * Adds a delivery to the delivery book.
+ */
+public class DeliveryAddCommand extends DeliveryCommand {
+
+    public static final String COMMAND_WORD = "add-d";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a delivery to the delivery book. "
+            + "Parameters: "
+            + PREFIX_NAME + "NAME "
+            + PREFIX_PHONE + "QUANTITY "
```
Comment (Wincenttjoi) [inline src/main/java/seedu/address/logic/commands/deliverycommand/DeliveryAddCommand.java]:
```suggestion
            + PREFIX_PHONE + "PHONE "
```

File: src/main/java/seedu/address/logic/commands/deliverycommand/DeliveryAddCommand.java (hunk 1 above)
Comment (Choo-Xing-Yu) [reply src/main/java/seedu/address/logic/commands/deliverycommand/DeliveryAddCommand.java]: HAHA i think we should just commit on his behalf. it's a minor issue

Comment (bobbylaw) [reply src/main/java/seedu/address/logic/commands/deliverycommand/DeliveryAddCommand.java]: My bad, Thanks for pointing it out hahah

**Comment by Wincenttjoi** [comment]
LGTM thanks!

**Comment by Wincenttjoi** [comment]
@zeranium97  U forgot to add a test for the add parser

image

In case you wanna do feel free hahah!

**Pull request merged**